engines/stark/stark.cpp

```diff
@@ -175,6 +175,8 @@ void StarkEngine::setStartupLocation() {
 }
 
 void StarkEngine::mainLoop() {
+	// Location change scripts require >0 loop duration
+	_global->setMillisecondsPerGameloop(10);
```

Comment:
Please don't hardcode the frame duration to 10 ms. If you need to make sure it's not zero, add a check in `StarkEngine::updateDisplayScene`.